docs/source/configuration/expanders.md

```diff
@@ -2,20 +2,24 @@
 
 You can configure the API expanders Volto uses using the `settings.apiExpanders` like this:
 
-```js
-import config from '@plone/volto/registry';
+```jsx
 import { GET_CONTENT } from '@plone/volto/constants/ActionTypes';
 
-config.settings.apiExpanders = [
```

Comment:
Good catch :)

Reply:
eh... it's like, I'm pointing people to the docs, but I don't like having to say "see that doc? Ignore what it says, do it like that"